code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -734,6 +744,58 @@
 /obj/item/clothing/head/hooded/winterhood/miner
 	icon_state = "winterhood_miner"
 
+/obj/item/clothing/suit/hooded/wintercoat/cosmic/true
+	name = "true cosmic winter coat"
+	icon_state = "coatcosmic"
+	item_state = "coatcosmic"
+	desc = "A true cosmic winter coat, it protects even from the rigors of space and comes with an inbuilt invisible jetpack."
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/cosmic/true
+	var/obj/item/tank/jetpack/suit/jetpack = /obj/item/tank/jetpack/suit/cosmiccoat

```

Comment:
The aforementioned hacky code requires this var to have the eaxct same name of the hardsuit counterpart